esmvalcore/cmor/_fixes/native6/era5.py

```diff
@@ -337,6 +457,19 @@ def fix_metadata(self, cubes):
         return cubes
 
 
+class Toz(Fix):
+    """Fixes for toz."""
+
+    def fix_metadata(self, cubes):
+        """Convert 'kg m-2' to 'm'."""
+        for cube in cubes:
+            cube.units = "kg m-2"
```

Comment:
Same comment here as in https://github.com/ESMValGroup/ESMValCore/pull/2178/files#r1790469038